lib/rules/length-zero-no-unit/__tests__/index.mjs

```diff
@@ -990,3 +990,31 @@ testRule({
 		},
 	],
 });
+
+testRule({
+	ruleName,
+	customSyntax: 'postcss-scss',
```

Comment:
[suggestion] If possible, we'd like to prefer standard syntaxes over non-standard ones in the codebase. How about testing with standard syntaxes like `@media (height > 0px) {}` instead of the SCSS syntax?

Reply:
Shall we:

- add another test case for `media` without custom syntax
- change the current test case to only use `/^mix/`

We do prefer standard syntaxes, but we also typically include non-standard test cases when the [original request](https://github.com/stylelint/stylelint/issues/8405) is in aid of supporting non-standard syntax.

This approach will let us do both.

Reply:
@jeddy3 @ybiquitous
Thanks for the review! I've fixed it.
[Add media query test case and example](https://github.com/stylelint/stylelint/pull/8472/commits/72bb54dc17c5363d48047a1ef80332b1222d8fd1)